src/python/CMakeLists.txt

```diff
@@ -1,21 +1,42 @@
+# Set SWIG policies
+cmake_policy(SET CMP0078 NEW)
```

Comment:
what are these?

Reply:
Something a bit more informative like

```
# Generate standard target names.
cmake_policy(...)
# Honor SWIG_MODULE_NAME via -module flag.
cmake_policy(...)
```

Reply:
These are documented in the [UseSWIG CMake docs](https://cmake.org/cmake/help/latest/module/UseSWIG.html):

## CMP0078:

[swig_add_library](https://cmake.org/cmake/help/latest/module/UseSWIG.html#command:swig_add_library):
Changed in version 3.13: This command creates a target with the specified <name> when policy [CMP0078](https://cmake.org/cmake/help/latest/policy/CMP0078.html#policy:CMP0078) is set to NEW. Otherwise, the legacy behavior will choose a different target name and store it in the SWIG_MODULE_<name>_REAL_NAME variable.

## CMP0086:

[Properties on Source Files](https://cmake.org/cmake/help/latest/module/UseSWIG.html#properties-on-source-files):

SWIG_MODULE_NAME
Specify the actual import name of the module in the target language. This is required if it cannot be scanned automatically from source or different from the module file basename. For example:

set_property(SOURCE mymod.i PROPERTY SWIG_MODULE_NAME mymod_realname)
Changed in version 3.14: If policy [CMP0086](https://cmake.org/cmake/help/latest/policy/CMP0086.html#policy:CMP0086) is set to NEW, -module <module_name> is passed to SWIG compiler.


I can't get the build to work without setting CMP0078 and CMP0086 to new. Since you are planning to migrate away from SWIG, I would recommend leaving this as is for now as they will be removed as part of that migration.